frontend/wallet/src/render/views/transactions/Transactions.re

```diff
@@ -102,6 +97,18 @@ type extractedResponse = {
   pending: array(TransactionCell.Transaction.t),
 };
 
+let gqlUserCommandToRecord = (userCommand, maybeDate) =>
+  TransactionCell.Transaction.UserCommand({
+    TransactionCell.Transaction.PaymentDetails.isDelegation:
```

Comment:
why is this key `TransactionCell.Transaction.PaymentDetails.isDelegation:` instead of `isDelegation`?

Reply:
the isDelegation is defined inside of `TransactionCell.Transaction.PaymentDetails`. We could probably do this cleaner by locally opening TransactionCell.Transaction

Reply:
why do we do it for just the first key, and not the others?